CHANGELOG.md

```diff
@@ -82,7 +85,6 @@
 * Fixed an issue where **validate** did not properly check `conf.json` when the latter is modified.
 * Fixed an issue in the **upload** command, where a prompt was not showing on the console.
 * Fixed an issue where running **lint** failed installing dependencies in containers.
-* Added integration configuration for *Cortex REST API* integration.
```

Comment:
is this moved to `1.14.4` purposely?

Reply:
It was by mistake merged and written under 1.12.0. It is related to the new release.